Documentation/Developer/InterlinkInventories.rst

```diff
@@ -0,0 +1,102 @@
+
+The preferred version can be set in the guides.xml to `dev`, `stable`
+`oldstable` or a specific minor version, for example `8.7`.
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+    >
+        <project title="Render guides"/>
+        <extension
+            class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+            typo3-core-preferred="stable"
+        />
+    </guides>
+
+or
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml, excerpt
+
+    <extension
+        class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+        typo3-core-preferred="8.7"
+
+It is not necessary anymore to list each of the standard inventories in the
+guides.xml anymore. If desired you can override or redefine standard interlink
+inventories or define new ones:
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+    >
+        <project title="Render guides"/>
+        <extension
+            class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+            typo3-core-preferred="stable"
+        />
+        <!-- explicitly link to version 8.7 of TYPO3 Explained -->
+        <inventory id="t3coreapi_v8" url="https://docs.typo3.org/m/typo3/reference-coreapi/8.7/en-us/"/>
+        <!-- ext_sys_note and ext_sys_note_stable should always link to main-->
+        <inventory id="ext_sys_note" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
+        <inventory id="ext_sys_note_stable" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
+    </guides>
+
+The following links:
+
+..  code-block:: rst
+
+    *   :ref:`TYPO3 Explained, always version 8.7 <t3coreapi_v8:start>`
+    *   :ref:`Sys note always goes to main <ext_sys_note:start>`
+    *   :ref:`Sys note always goes to main <ext_sys_note_stable:start>`
+
+*   :ref:`TYPO3 Explained, always version 8.7 <t3coreapi_v8:start>`
+*   :ref:`Sys note always goes to main <ext_sys_note:start>`
+*   :ref:`Sys note always goes to main <ext_sys_note_stable:start>`
+
+
+Adding a new TYPO3 version or manual
+====================================
+
+In the event of a change in long-term support, adjustments to the corresponding
+TYPO3 versions can be made directly in the theme within the enum
+:php:`\T3Docs\Typo3DocsTheme\Inventory\Typo3VersionMapping`.
```

Comment:
The backslashes must be escaped now, which is be a breaking change to the Sphinx-based rendering. Currently it look like:

![Bildschirmfoto vom 2023-12-26 15-02-41](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/e53e9dcb-a5be-4943-8891-a92df62537d9)

Is this known? I assume, this is a bug?

Reply:
I will make an issue and fix it in the renderer. I didnt notice yet. We don't have to change it here as the renderer fix will fix that